src/main.py

```diff
@@ -34,17 +34,25 @@ class Telemetry(metaclass=SingletonMetaClass):
     application name, version and tracking id just once. Later the instance can be created without parameters.
     """
     def __init__(self, app_name: str = None, app_version: str = None, tid: str = None,
-                 backend: [str, None] = 'ga'):
+                 backend: [str, None] = 'ga', enable_opt_in_dialog=True):
```

Comment:
please add description for this new flag in the comments

Reply:
Done.